webviz_subsurface/plugins/_well_completions.py

```diff
@@ -30,8 +35,9 @@ class WellCompletions(WebvizPluginABC):
     * **`ensembles`:** Which ensembles in `shared_settings` to visualize.
     * **`compdat_file`:** csvfile with compdat data per realization
     * **`zone_layer_mapping_file`:** Lyr file specifying the zone->layer mapping \
+    * **`stratigraphy_file`:** Json file defining the stratigraphic levels \
     * **`well_attributes_file`:** Json file with categorical well attributes \
```

Comment:
JSON is all capital, alternatively we can perhaps write "`.json` file" (including the backtick formatting). If so, we could also on the lines above write `.lyr` file, `.csv` file etc.